documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -354,7 +354,7 @@ When referencing an empty file, all metrics are exposed as long as they have not
 [source,yaml,subs="+attributes"]
 ----
 kind: ConfigMap
-apiVersion: {KafkaApiVersion}
+apiVersion: v1
```

Comment:
We use a few _v1_ versions in the guides. For secrets, pods, service, volumes and so on. Does it make sense to use an attribute?

Reply:
It could make sense but because it impacts other files, let's do it in a different PR.

Reply:
Even if, thinking more, it's not something that is going to change. It's different from Strimzi version where we are in beta1 then beta2, v1, ... With ConfigMap the v1 version is well defined, as for Secrets, and so on. @scholzj wdyt?

Reply:
If you make this attribute, you need to have many attributes ... SecretApiVersion, ConfigMapAPiVersion, ServiceApiVersion etc. as they might change independently. So you would end up with many attributes for something what changes rarely if ever. IMHO not worth it. (the issue here is typo in the example - not something what changed)